hypothesis-python/docs/compatibility.rst

```diff
@@ -86,24 +86,24 @@ test to take this form, or ask the framework maintainer to support our
 
 In terms of what's actually *known* to work:
 
-  * Hypothesis integrates as smoothly with pytest and unittest as we can make it,
-    and this is verified as part of the CI.
-  * :pypi:`pytest` fixtures work in the usual way for tests that have been decorated
-    with :func:`@given <hypothesis.given>` - just avoid passing a strategy for
-    each argument that will be supplied by a fixture.  However, each fixture
-    will run once for the whole function, not once per example.  Decorating a
-    fixture function with :func:`@given <hypothesis.given>` is meaningless.
-  * The :func:`python:unittest.mock.patch` decorator works with
-    :func:`@given <hypothesis.given>`, but we recommend using it as a context
-    manager within the decorated test to ensure that the mock is per-test-case
-    and avoid poor interactions with Pytest fixtures.
-  * Nose works fine with Hypothesis, and this is tested as part of the CI. ``yield`` based
-    tests simply won't work.
-  * Integration with Django's testing requires use of the :ref:`hypothesis-django` extra.
-    The issue is that in Django's tests' normal mode of execution it will reset the
-    database once per test rather than once per example, which is not what you want.
-  * :pypi:`coverage` works out of the box with Hypothesis; our own test suite has
-    100% branch coverage.
+* Hypothesis integrates as smoothly with pytest and unittest as we can make it,
+  and this is verified as part of the CI.
+* :pypi:`pytest` fixtures work in the usual way for tests that have been decorated
+  with :func:`@given <hypothesis.given>` - just avoid passing a strategy for
+  each argument that will be supplied by a fixture.  However, each fixture
+  will run once for the whole function, not once per example.  Decorating a
+  fixture function with :func:`@given <hypothesis.given>` is meaningless.
+* The :func:`python:unittest.mock.patch` decorator works with
+  :func:`@given <hypothesis.given>`, but we recommend using it as a context
+  manager within the decorated test to ensure that the mock is per-test-case
+  and avoid poor interactions with Pytest fixtures.
+* Nose works fine with Hypothesis, and this is tested as part of the CI. ``yield`` based
```

Comment:
[nose is unmaintained](https://nose.readthedocs.io/en/latest/) at this point; maybe we should switch over to testing against [successor project `nose2`](https://docs.nose2.io/en/latest/)?  It's not super-popular but it'd be easy enough to port that one CI job (not in this PR)

Reply:
unrelated comment to any CI testing: I think I dropped nose references in the docs on the original docs PR, and would still support doing so. usage in the latest jetbrains survey was something like 2%, and dropping year on year?